aiounifi/models/device.py

```diff
@@ -722,6 +723,49 @@ def create(
         )
 
 
+@dataclass
+class DeviceSetLedStatus(ApiRequest):
+    """Request object for setting LED status of device."""
+
+    @classmethod
+    def create(
+        cls,
+        device: "Device",
+        status: str = "on",
+        brightness: int | None = None,
+        color: str | None = None,
+    ) -> Self:
+        """Set LED status of device."""
+
+        data = {"led_override": status}
+        if device.supports_led_ring:
+            # Validate brightness parameter
+            if brightness is not None and (0 <= brightness <= 100):
+                data["led_override_color_brightness"] = brightness
+            else:
+                raise AttributeError("Brightness must be within the range [0, 100].")
+
+            # Validate color parameter
+            if color is not None and re.match(r"^#(?:[0-9a-fA-F]{3}){1,2}$", color):
```

Comment:
Same as brightness need to be reworked if you support either one option to be sent.